tests/playwright/utils/global-utils.js

```diff
@@ -0,0 +1,24 @@
+// Playwright-dev-page.js
+const { expect } = require( '@playwright/test' );
+
+exports.GlobalUtils = class GlobalUtils {
+    constructor( page, url ) {
+        // Prod Selectors
+        this.url = url;
+        this.page = page;
+    }
+
+    async checkTextIsPresentInArray( page, selector, testData ) {
+        const featuresList = await selector.allTextContents();
+        for ( let i = 0; i < testData.length; i++ ) {
```

Comment:
selector might return an empty array - is that ok ?

Reply:
not sure. I'll add a check for it.